features/networking/sctp.feature

```diff
@@ -26,6 +33,12 @@ Feature: SCTP related scenarios
       | ["spec"]["nodeName"]      | <%= cb.workers[1].name %> |
     Then the step should succeed
     And the pod named "sctpclient" becomes ready
+    And I wait up to 20 seconds for the steps to pass:
```

Comment:
@weliang1 @anuragthehatter I see a lot of duplication off this step.  It might be helpful for others to encapsulate the logic into a step that others can reuse.  Perhaps a step like, `Given there is the command #{QUOTED} in the #{QUOTED} pod`

Reply:
@pru New commit remove above steps, no need to create this